### ISSUE TITLE
A11y audit & update scorecard after v6

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-org#3618]
v6 is live, now that we are publishing org's a11y scorecard, we are due for an updated scorecard and a11y audit.

- [x] update the a11y config to make sure it is optimized for the new IA
- [x] note all a11y violations, address low hanging fruit, document further work needed
- [x] fill out new scorecard and update the website page to reflect new report
